RiotSharp/Misc/DateTimeConverterFromStringTimestamp.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace RiotSharp
+{
+    class DateTimeConverterFromStringTimestamp : JsonConverter
```

Comment:
This seems awfully similar to [`DateTimeConverterFromLong`](https://github.com/BenFradet/RiotSharp/blob/develop/RiotSharp/Misc/DateTimeConverterFromLong.cs), is this intentional?